Doc/library/sys.rst

```diff
@@ -1581,8 +1621,8 @@ always available.
 
    .. versionchanged:: 3.7
 
-      ``'opcode'`` event type added; :attr:`f_trace_lines` and
-      :attr:`f_trace_opcodes` attributes added to frames
+      ``'opcode'`` event type added; :attr:`!f_trace_lines` and
+      :attr:`!f_trace_opcodes` attributes added to frames
```

Comment:
```suggestion
      :attr:`!f_trace_opcodes` attributes added to :ref:`frames <frame-objects>`
```

Reply:
Yeah, I considered this one, but deliberately left it out, as it was only the "version changed" for the `f_trace_opcodes` mentioned earlier, which we already added a frame object link to, so I concluded there was no need for a link here as well :)